### PR TITLE
Add missing translation domains

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x 
| Description?  | Add missing translation domains
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Please check that Import migrated page is alright as well as Database Manager migrated page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13605)

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]
Code looks legit, waiting for wording.

File: src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php]: This one seems weird.. I think it's the second parameters array which is missing

```suggestion
            new EntityField('id', $this->trans('ID', [], 'Admin.Global')),
```

File: src/Core/Import/EntityField/Provider/CustomerFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/CustomerFieldsProvider.php]:
```suggestion
            new EntityField('id', $this->trans('ID', 'Admin.Global')),
```

File: src/Core/Import/EntityField/Provider/AddressFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/AddressFieldsProvider.php:59]: Is it really useful since the default domain of this class' `trans` method is `Admin.Advparameters.Feature` ?

File: src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php:58]: wouldn't it be more logical to use `Admin.Catalog.Feature` ?

File: src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php:57]: `Admin.Advparameters.Feature` is already the default domain in this class, although something linked to `Product` would seem more logical

File: src/Core/Import/EntityField/Provider/ProductFieldsProvider.php
Comment (jolelievre) [inline src/Core/Import/EntityField/Provider/ProductFieldsProvider.php]:
```suggestion
            new EntityField('id', $this->trans('ID', 'Admin.Global')),
```

File: src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php
Comment (jolelievre) [reply src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php]: My bad, there is no array parameter in this case, however the domain parameter was already provided

**Review by jolelievre — DISMISSED** [review]
I didn't comment them all but a few calls have too many parameters (domain already present)
And I'm wondering about the use of `Admin.Advparameters.Feature` averywhere? Is this because it's for the Import feature? Maybe some of these keys already exist in other domain nonetheless

File: src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php
Comment (PierreRambaud) [reply src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php]: yep, because of https://github.com/PrestaShop/PrestaShop/blob/7ab0cae901f6b932414415fa173c5802818f6ee6/src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php#L92 (in case someone see your comments)

**Review by PierreRambaud — DISMISSED** [review]
Because @jolelievre has a better eyes :smile:

File: src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php
Comment (matks) [reply src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php:57]: > I didn't comment them all but a few calls have too many parameters (domain already present)

Yes, that's a mistake indeed. I'm going to fix t.

> And I'm wondering about the use of `Admin.Advparameters.Feature` averywhere? Is this because it's for the Import feature? Maybe some of these keys already exist in other domain nonetheless

I did it because this is the domain being used, as you can see from https://github.com/PrestaShop/PrestaShop/blob/7ab0cae901f6b932414415fa173c5802818f6ee6/src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php#L90
```
/**
     * A shorter name method for translations.
     *
     * @param string $id translation ID
     * @param string $domain translation domain
     *
     * @return string
     */
    private function trans($id, $domain = 'Admin.Advparameters.Feature')
    {
        return $this->translator->trans($id, [], $domain);
    }
```

This private `trans()` saves a few developer time but it cannot be parsed by the Translations Extraction tool. That's why I added the domains manually.

Comment (jolelievre) [reply src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php:57]: Oh I see that's why you had to add the domains everywhere. Then you should probably remove the default value in the `trans` methods to avoid misunderstanding and missing domains.
That said, I know it was used up until now but I'm still wondering if `Admin.Advparameters.Feature` is the appropriate domain in all those cases, we should wait for @LouiseBonnard approval.

**Comment by jolelievre** [comment]
> Because @jolelievre has a better eyes 😄

My eyes were WIDE open after the last game of throne's episode..

**Comment by matks** [comment]
> > Because @jolelievre has a better eyes 😄
> 
> My eyes were WIDE open after the last game of throne's episode..

🐉 ⚔️

**Comment by matks** [comment]
Fixed the domain duplicates, now waiting for wording validation

File: src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php
Comment (LouiseBonnard) [reply src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php:58]: We try to avoid duplicate the strings...

File: src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php
Comment (LouiseBonnard) [inline src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php]:
```suggestion
            new EntityField('product_reference', $this->trans('Product reference', 'Admin.Advparameters.Feature')),
```

File: src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php
Comment (LouiseBonnard) [inline src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php]:
```suggestion
            new EntityField('note', $this->trans('Note', 'Admin.Global')),
```

**Comment by matks** [comment]
@LouiseBonnard applied your changes
@jolelievre please re-review 😄

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matks

**Pull request merged**
